tests/jobs/test_backfill_job.py

```diff
@@ -929,8 +960,11 @@ def run_backfill(cond):
                     dag_maker.create_dagrun(
                         # Existing dagrun that is not within the backfill range
                         run_id=run_id,
+                        state=State.RUNNING,
                         execution_date=DEFAULT_DATE + datetime.timedelta(hours=1),
+                        start_date=DEFAULT_DATE,
```

Comment:
dag_maker has a default state=`State.RUNNING` and start_date=DEFAULT_DATE. Why are we adding this? Less code is better...
I don't think you should change this test

Reply:
Yes,I will resolve these problems, thank you